test/e2e/auth/service_accounts.go

```diff
@@ -520,6 +521,105 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 			framework.Failf("Unexpected error: %v\n%s", err, logs)
 		}
 	})
+
+	ginkgo.It("should support OIDC discovery of service account issuer [Feature:ServiceAccountIssuerDiscovery]", func() {
```

Comment:
let's split the agnhost and e2e test into a separate PR... need to sync up with testing on how to build/push agnhost updates

Reply:
Coming Soon!

Reply:
Agnhost: https://github.com/kubernetes/kubernetes/pull/88049
Test: https://github.com/kubernetes/kubernetes/pull/88048